test/spec/modules/mobfoxBidAdapter_spec.js

```diff
@@ -2,161 +2,126 @@ describe('mobfox adapter tests', function () {
-    });
-
-    it('should add empty bid responses if no bids returned', function () {
-      let bidderRequest = {
-        bidderCode: 'mobfox',
-        bids: [
-          {
-            bidId: 'bidId1',
-            bidder: 'mobfox',
-            params: {},
-            sizes: [[300, 250]],
-            placementCode: 'test-gpt-div-1234'
-          }
-        ]
-      };
-
-      // empty ads in bidresponse
-      let requestParams = utils.cloneJson(mockRequestsParams.banner);
-      requestParams.adspace_width = 1231564; // should return an error
-      bidderRequest.bids[0].params = requestParams;
-      $$PREBID_GLOBAL$$._bidsRequested.push(bidderRequest);
-      // adapter needs to be called, in order for the stub to register.
-      adapter().callBids(bidderRequest);
-
-      let bidPlacementCode1 = stubAddBidResponse.getCall(0).args[0];
-      let bidResponse1 = stubAddBidResponse.getCall(0).args[1];
-      expect(bidPlacementCode1).to.equal('test-gpt-div-1234');
-      expect(bidResponse1.getStatusCode()).to.equal(CONSTANTS.STATUS.NO_BID);
-      expect(bidResponse1.bidderCode).to.equal('mobfox');
-    });
+  const bidRequest = [{
+    code: 'div-gpt-ad-1460505748561-0',
+    sizes: [[320, 480], [300, 250], [300, 600]],
+    // Replace this object to test a new Adapter!
+    bidder: 'mobfox',
+    bidId: '5t5t5t5',
+    params: {
+      s: '267d72ac3f77a3f447b32cf7ebf20673', // required - The hash of your inventory to identify which app is making the request,
+      imp_instl: 1 // optional - set to 1 if using interstitial otherwise delete or set to 0
+    },
+    placementCode: 'div-gpt-ad-1460505748561-0',
+    requestId: 'c241c810-18d9-4aa4-a62f-8c1980d8d36b',
+    transactionId: '31f42cba-5920-4e47-adad-69c79d0d4fb4'
+  }];
+
+  const bidRequestInvalid1 = [{
+    code: 'div-gpt-ad-1460505748561-0',
+    sizes: [[320, 480], [300, 250], [300, 600]],
+    // Replace this object to test a new Adapter!
+    bidder: 'mobfox',
+    bidId: '5t5t5t5',
+    params: {
+      imp_instl: 1 // optional - set to 1 if using interstitial otherwise delete or set to 0
+    },
+    placementCode: 'div-gpt-ad-1460505748561-0',
+    requestId: 'c241c810-18d9-4aa4-a62f-8c1980d8d36b',
+    transactionId: '31f42cba-5920-4e47-adad-69c79d0d4fb4'
+  }];
+
+  const bidRequestInvalid2 = [{
+    code: 'div-gpt-ad-1460505748561-0',
+    sizes: [[320, 480], [300, 250], [300, 600]],
+    // Replace this object to test a new Adapter!
+    bidder: 'mobfox',
+    bidId: '5t5t5t5',
+    params: {
+      s: '267d72ac3f77a3f447b32cf7ebf20673', // required - The hash of your inventory to identify which app is making the request,
+      imp_instl: 1 // optional - set to 1 if using interstitial otherwise delete or set to 0
+    },
+    placementCode: 'div-gpt-ad-1460505748561-0',
+    transactionId: '31f42cba-5920-4e47-adad-69c79d0d4fb4'
+  }];
+
+  describe('test valid MF request success', function () {
+    let isValid = adapter.spec.isBidRequestValid(bidRequest[0]);
```

Comment:
Test cases need to be in `it` to be executed. As of now none of the tests are executed.
This https://gist.github.com/samwize/8877226 will be helpful